src/components/App.tsx

```diff
@@ -213,6 +214,17 @@ export default class App extends PureComponent<Props, State> {
 		});
 	};
 
+	focus = (id: string): void => {
+		this.setState(previousState => {
+			const targetIdOrNext =
+				previousState.focusables.find(focusable => focusable.id === id)?.id ||
+				this.findNextFocusable(previousState);
```

Comment:
Curious about this behavior, why do you think it should focus the next element if no element by ID is found? I'd think it would do nothing.

Reply:
Fair point, probably the expected behavior! I'll remove.

Reply:
Done